src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3809.cs

```diff
@@ -1,67 +1,69 @@
 ﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
 using UITest.Appium;
 using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues
 {
 	public class Issue3809 : _IssuesUITest
 	{
+		const string SetPagePadding = "Set Page Padding";
+		const string SafeAreaText = "Safe Area Enabled: ";
+		const string PaddingLabel = "paddingLabel";
+		const string SafeAreaAutomationId = "SafeAreaAutomation";
+
 		public Issue3809(TestDevice testDevice) : base(testDevice)
 		{
 		}
 
 		public override string Issue => "SetUseSafeArea is wiping out Page Padding ";
 
-		// TODO: The _ variables need to be AutomationId values
-		//void AssertSafeAreaText(string text)
-		//{
-		//	var element =
-		//		RunningApp
-		//			.WaitForFirstElement(_safeAreaAutomationId);
-
-		//	element.AssertHasText(text);
-		//}
-
-		//[Test]
-		//[Category(UITestCategories.Layout)]
-		//public void SafeAreaInsetsBreaksAndroidPadding()
-		//{
-		//	// ensure initial paddings are honored
-		//	AssertSafeAreaText($"{_safeAreaText}{true}");
-		//	var element = App.WaitForFirstElement(_paddingLabel);
+		void AssertSafeAreaText(string text)
+		{
+			var element = App.WaitForFirstElement(SafeAreaAutomationId);
 
-		//	bool usesSafeAreaInsets = false;
-		//	if (element.ReadText() != "25, 25, 25, 25")
-		//		usesSafeAreaInsets = true;
+			ClassicAssert.AreEqual(element.GetText(), text);
+		}
 
-		//	Assert.AreNotEqual(element.ReadText(), "0, 0, 0, 0");
-		//	if (!usesSafeAreaInsets)
-		//		Assert.AreEqual(element.ReadText(), "25, 25, 25, 25");
+		[Test]
+		[Category(UITestCategories.Layout)]
```

Comment:
**Step 3**: Add Category. Without a category, the project test project will not compile.

If the category is relevant to all tests in a class, you can also put it on the class.